src/styles/theme/oc-sidebar.scss

```diff
@@ -0,0 +1,111 @@
+.oc-sidebar {
+  display: flex;
+  flex-flow: column wrap;
+  width: $width-medium;
+  height: 100%;
+  padding: $gutter 0;
+  max-height: 100%;
+  background-color: $primary-background;
+  overflow-y: auto;
+  box-sizing: border-box;
+
+  &-logo {
+    padding: 0 $small-gutter;
+    margin-bottom: $large-margin;
+    text-align: center;
+
+    &:hover,
+    &:focus {
+      text-decoration: none;
+    }
+
+    &-img {
+      @extend .uk-width-small;
+    }
+
+    &-text {
+      @extend .uk-text-lead;
+      @extend .oc-light;
+    }
+  }
+
+  &-upper-content {
+    padding: 0 $small-gutter $gutter;
+  }
+
+  &-nav {
+    flex: 1;
+
+    & > ul {
+      padding: 0;
+      margin: 0;
+    }
+  }
+
+  &-nav-item {
+    transition: background-color 0.3s ease-in-out;
+
+    &:hover,
+    &:focus {
+      background-color: darken($primary-background, 5);
+    }
+
+    & > a,
+    & > a > .oc-icon > svg {
+      @extend .oc-light;
+    }
+
+    & > a {
+      display: flex;
+      flex-flow: row wrap;
+      padding: $small-gutter;
+      align-items: center;
+
+      &:hover,
+      &:focus {
+        text-decoration: none;
+      }
+    }
+
+    & .oc-icon {
+      margin-right: $small-margin;
+    }
+
+    &.uk-active {
+      background-color: darken($primary-background, 5);
+
+      & > a {
+        font-weight: bold;
+
+        &:hover {
+          cursor: default;
+        }
+      }
+    }
+  }
+
+  &-footer {
+    padding: 0 $small-gutter;
+  }
+
+  &-fixed {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+  }
+
+  &-fixed &-button-close {
+    position: absolute;
+    top: $small-gutter;
+    right: $small-gutter;
+
+    & .oc-icon > svg {
+      @extend .oc-light;
+    }
+  }
+
+  &-fixed &-logo {
+    margin-top: $medium-margin;
+  }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5442879)